js/plugins/iq.js

```diff
@@ -129,6 +141,13 @@ var IqPlugin= function (framework)
       if (metadata.metadata.enabled != null){
         iqEnabled = metadata.metadata.enabled;
       }
+      // Are we possibly sending thrift events as well? If so we do not want to send
+      // any duplicate events with analytics.js here, only new events not reported by
+      // thrift in core with ooyala_analytics.js 
+      if (metadata.metadata.allowThrift != null){
```

Comment:
Should we also check that the metadata.metadata.allowThrift is not undefined before setting allowThrift?

Reply:
@aeng7 would undefined even be possible here? We know that metadata and metadata.metadata both exist at this point, so wouldn't metadata.metadata.allowThrift only return null if there is no value for that key in the object? If it can we would also need to add a check for enabled above as well...

Reply:
yep, if the allowThrift key does not exist, or exists but has value undefined, the value of allowThrift will have value undefined.

Reply:
@aeng7 ok, fixed